src/utils/hub-channel.js

```diff
@@ -54,6 +58,21 @@ export default class HubChannel extends EventTarget {
     return this.can(permission);
   }
 
+  canEnterRoom(hub) {
+    if (!hub) return false;
+    if (this.canOrWillIfCreator("update_hub")) return true;
+
+    const roomEntrySlotCount = Object.values(this.presence.state).reduce((acc, { metas }) => {
```

Comment:
see the reticulum PR, i'd refactor 'room entry slots' to be 'room member counts'